src/pages/index.js

```diff
@@ -69,74 +69,12 @@ const moreLinks = [
 
 const utmParameters = `?utm_source=starter&utm_medium=start-page&utm_campaign=default-starter`
```

Comment:
Is there a reason why these links are here? not sure if they are helpful or distracting. Seems like they added it to the starter. You can get rid of them too.